Rp20_1e/README.md

```diff
@@ -1,9 +1,12 @@
-#A Character Sheet for Rp20 (v1)
+# A Character Sheet for Rp20 (v1)
 
 A simple roleplaying system meant to be dynamic and expansible, for GM’s and players that that enjoy customization.
 This character sheet is meant for use on Roll20.net, and is meant to be available to all users regardless of subscription level.
 
-##Changelog
+## Changelog
 
-###2017-12-11 - v1.0
+### 2017-12-12 - v1.0.1
```

Comment:
Also, kinda figured that it would make sense to use Semantic Version Numbering https://semver.org/